src/include/daos_cont.h

```diff
@@ -671,6 +670,48 @@ int
 daos_cont_open2(daos_handle_t poh, const char *cont, unsigned int flags,
 		daos_handle_t *coh, daos_cont_info_t *info, daos_event_t *ev);
 
+int
+daos_cont_destroy2(daos_handle_t poh, const char *cont, int force,
+		   daos_event_t *ev);
+
+#if defined(__cplusplus)
+}
+
+#define daos_cont_open daos_cont_open_cpp
+static inline int
+daos_cont_open_cpp(daos_handle_t poh, const char *cont, unsigned int flags, daos_handle_t *coh,
+		   daos_cont_info_t *info, daos_event_t *ev)
+{
+	return daos_cont_open2(poh, cont, flags, coh, info, ev);
+}
+
+static inline int
+daos_cont_open_cpp(daos_handle_t poh, const uuid_t cont, unsigned int flags, daos_handle_t *coh,
+	       daos_cont_info_t *info, daos_event_t *ev)
+{
+	char str[37];
+
+	uuid_unparse(cont, str);
+	return daos_cont_open2(poh, str, flags, coh, info, ev);
+}
+
+#define daos_cont_destroy daos_cont_destroy_cpp
+static inline int
+daos_cont_destroy_cpp(daos_handle_t poh, const char *cont, int force, daos_event_t *ev)
+{
+	return daos_cont_destroy2(poh, cont, force, ev);
+}
+
+static inline int
+daos_cont_destroy_cpp(daos_handle_t poh, const uuid_t cont, int force, daos_event_t *ev)
+{
+	char str[37];
```

Comment:
what is this magic 37?

Reply:
uuid len

Reply:
I copied from Johann.   Looks like there is a macro for it though

UUID_STR_LEN is defined as 37

Reply:
From the `uuid_parse` docs:
```
The uuid_unparse function converts the supplied UUID uu from the binary representation into a 36-byte string (plus trailing '\0')
```

It's the fixed-format UUID text representation as defined in https://datatracker.ietf.org/doc/html/rfc4122